docs/user_api/index.rst

```diff
@@ -54,9 +54,16 @@ Remapping
 ---------
 .. autosummary::
    :toctree: generated/
+   :template: autosummary/accessor.rst
+
+   UxDataArray.remap
+
+.. autosummary::
+   :toctree: generated/
+   :template: autosummary/accessor.rst
 
-   UxDataset.nearest_neighbor_remap
-   UxDataset.inverse_distance_weighted_remap
+   UxDataset.remap.nearest_neighbor_remap
```

Comment:
@philipc2 This isn't showing up in the user api on the docs, any idea why this might be happening?

Reply:
Think it might be a small typo. The functions are `.remap.nearest_neighbor` and `.remap.inverse_distance_weighted`, without the `_remap` suffix

Reply:
Oh, yeah that is a good catch thanks.